src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -37,7 +40,12 @@ export const UserAvatar: React.FC<UserAvatarProps> = ({
     "user-avatar--clickable": onClick !== undefined,
   });
 
+  const status = user?.status ?? "";
+
   const imageClasses = classNames("user-avatar__image", imageClassName);
+  const statusClasses = classNames("user-avatar__status-dot", statusClassName, {
+    [`user-avatar__status-dot--${status}`]: status,
+  });
```

Comment:
This logic should be refactored to use `isOnline` and `status`. Eg. show status only if isOnline(which makes sense)